ocs_ci/ocs/resources/packagemanifest.py

```
@@ -307,3 +308,93 @@ def get_selector_for_ocs_operator():
+        or not filtered_output.strip()
+        or filtered_output.strip() == "null"
+    ):
+        raise ValueError(
+            f"No packagemanifest found for '{package_name}' with "
+            f"catalogSource '{catalog_source}'"
+        )
+
+    packagemanifest_data = yaml.safe_load(filtered_output)
+
+    if not packagemanifest_data:
+        raise ValueError(f"Failed to parse packagemanifest data for '{package_name}'")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)